pennylane/devices/null_qubit.py

```diff
@@ -11,320 +11,424 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-r"""
-The null.qubit device is a no-op device for benchmarking PennyLane's auxiliary functionality outside direct circuit evaluations.
 """
-from collections import defaultdict
+The null.qubit device is a no-op device for benchmarking PennyLane's
+auxiliary functionality outside direct circuit evaluations.
```

Comment:
We should update this to mention that `null.qubit` is not just for internal testing but also for resource estimation

Reply:
updated, lmk what you think

Reply:
Nice, thanks!